Wireframe/index.html

```diff
@@ -3,11 +3,83 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <title>DELLA BELLA WIREFRAME</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Cabin:wght@400;700&family=Raleway:wght@100;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-</body>
-</html
\ No newline at end of file
+
+      <!-- NAV -->
+       <nav>
+        <ul>
```

Comment:
When you ran Lighthouse on this website, what did it tell you?

Reply:
<img width="1214" alt="Screenshot 2024-10-21 at 22 15 44" src="https://github.com/user-attachments/assets/4dc7870f-97f1-4ee7-a6c9-6a1ec16dcee7">

I had to fix some Colour Contrast but the score looks good now.